src/view/filters.js

```diff
@@ -0,0 +1,33 @@
+import {createElement} from '../render.js';
+
+const createHeaderFilterButtonsTemplate = () => `<form class="trip-filters" action="#" method="get">
+<div class="trip-filters__filter">
+  <input id="filter-everything" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="everything" checked="">
+  <label class="trip-filters__filter-label" for="filter-everything">Everything</label>
+</div>
+
+<div class="trip-filters__filter">
+  <input id="filter-future" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="future">
+  <label class="trip-filters__filter-label" for="filter-future">Future</label>
+</div>
+
+<button class="visually-hidden" type="submit">Accept filter</button>
+</form>`;
```

Comment:
```suggestion
const createHeaderFilterButtonsTemplate = () => (
`<form class="trip-filters" action="#" method="get">
<div class="trip-filters__filter">
  <input id="filter-everything" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="everything" checked="">
  <label class="trip-filters__filter-label" for="filter-everything">Everything</label>
</div>

<div class="trip-filters__filter">
  <input id="filter-future" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="future">
  <label class="trip-filters__filter-label" for="filter-future">Future</label>
</div>

<button class="visually-hidden" type="submit">Accept filter</button>
</form>`
);
```

ну и пробелы соответственно можно расставить, так будет читабельней. Ну и естественно, так лучше написать везде))